kubernetes.go

```diff
@@ -39,8 +39,8 @@ type KubernetesInstalledApplication struct {
 	Configuration map[string]string `json:"configuration,omitempty"`
 }
 
-// KubernetesCluster is a Kubernetes k3s cluster
-type KubernetesCluster struct {
+// KubernetesItem is a Kubernetes item inside the cluster
+type KubernetesItem struct {
```

Comment:
This should be KubernetesClusterItem (a Kubernetes Item doesn't make sense)

Reply:
And in fact, having just viewed the whole file, this really should be a `KubernetesCluster`, there's no need for the `Item` - it purely represents a cluster. It's the list of clusters that should be changed.